Project/webpack.config.js

```diff
@@ -91,14 +129,14 @@ module.exports = {
             app.use(bodyParser.json());
             app.post('/getCommunicationUserToken', async (req, res) => {
                 try {
-                    const communicationUserIdentifier = await communicationIdentityClient.createUser();
-                    const communicationUserToken = await communicationIdentityClient.issueToken(communicationUserIdentifier, ["voip"]);
+                    const userId = await communicationIdentityClient.createUser();
+                    const communicationUserToken = await communicationIdentityClient.getToken(userId, ["voip"]);
```

Comment:
same as above

Reply:
So now what we get is two things,

communicationUserIdentifier, which is an object which has the user id.

And

CommunicationAccessToken which is an obj that has the token.

Hence i think we should make it nice and clear to return { communicationAccessToken, oneSignalRegisraionToken, communicationUserIdentifier } so that we dont make users have to think more

Reply:
Since these are things coming from an ACS lib, it would be nice to make the var names nice and clear